### DISCUSSION TITLE
Copilot does not use a successfully configured proxy for making requests.

### DISCUSSION THREAD

**Discussion by luoxiaofeng1029** [community/community#67663] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

In your documentation, I configured a proxy in the JetBrains IDE. However, by observing my VPN, I didn't find that Copilot's requests were going through the proxy. So, I tried configuring an incorrect proxy in the JetBrains IDE, and Copilot was still able to access normally. This indicates that even if the correct proxy is configured in IDEA, Copilot will not use the proxy for requests.

**Comment by Akash1134** [comment]
Hi @luoxiaofeng1029 ,

The proxy settings you configure in your IDE are only used for requests made by the IDE itself, such as downloading updates or accessing external resources. They do not affect the network traffic generated by Copilot.

**Reply by luoxiaofeng1029** [reply]
Thank you for your reply. As stated in the official documentation, you can set up a proxy for CoPilot by following the document's configuration.
[Configuring network settings for GitHub Copilot](https://docs.github.com/en/copilot/configuring-github-copilot/configuring-network-settings-for-github-copilot)

**Comment by luoxiaofeng1029** [comment]
who can help me,thanks

**Comment by luoxiaofeng1029** [comment]
who can help me,thanks

**Comment by luoxiaofeng1029** [comment]
who can help me,thanks
